### DISCUSSION TITLE
Readme 문서에 Shell 디렉토리 내용에 대한 설명을 넣어주시면 안되겠습니까.

### DISCUSSION THREAD

**Discussion by infinitekh** [junegunn/fzf#2488] [Ideas]
먼저 선생님의 프로그램을 잘 사용하고 있습니다. 매우 감사합니다. 

매번 설치할 때마다, **<Tab> 기능을 사용하는 셋팅을 하는데서 막히는 상황입니다. (구글링으로 외부에서 설명듣고 해결)

Readme에 분명 **<Tab>이나 Alt_C Ctrl_T 같은 설명들이 있는데, 그것이 작동하도록하기위한 제반에 대해서는 안나와있습니다. 

Shell 폴더에 bashrc나 profile수준에서 불러오도록 만들어진 파일이 있는거 보면, 준비는 되어 있습니다. 

어찌저찌해서 구글링을 여러번 해서 그런 파일들이 있는 것을 확인하고 지금은 잘 세팅을 한 상황입니다. 

새로운 컴퓨터에 OS를 깔고 Vim 셋팅을 하는건 보통 6개월에서 1년정도의 시간차가 있습니다. 

다시 그 상황을 겪을 때마다, 같은 부분에서 막힙니다. source fzf_completion.bash 같은 부분을 Readme에 넣어주세요.
$(zfz)로 그냥 사용해도 되기는 하지만, 분명히 멋진 기능이 있는 것 같은데 뭘 더해야 작동하는거지라는 고민이 사라지면 좋겠습니다. 

모든 사람들이 영어로 디스커션을 남기는데, 한국어로 글을 달아서 죄송합니다. 

불편하시다면, 지워주세요. 

좋은 프로그램 잘 사용하겠습니다.

**Comment by junegunn** [comment]
https://github.com/junegunn/fzf#installation
